CONTRIBUTING.rst

```diff
@@ -1,37 +1,163 @@
+
+   $ cd aiobotocore
+   $ virtualenv -p /usr/bin/python3 venv-aiobotocore
+   $ . venv-aiobotocore/bin/activate
 
-    $ git clone git@github.com:aio-libs/aiobotocore.git
+For standard python *venv-aiobotocore*:
 
-Create virtualenv with at least python3.5 (older version are not supported).
-For example using *virtualenvwrapper* commands could look like::
+.. code-block:: shell
 
    $ cd aiobotocore
-   $ mkvirtualenv --python=`which python3.5` aiobotocore
+   $ python3 -m venv venv-aiobotocore
+   $ . venv-aiobotocore/bin/activate
+
+For *virtualenvwrapper*:
+
+.. code-block:: shell
+
+   $ cd aiobotocore
+   $ mkvirtualenv -p /usr/bin/python3 venv-aiobotocore
+   $ workon venv-aiobotocore
+
+There are other tools like *pyvenv* but you know the rule of thumb
+now: create a python3 virtual environment and activate it.
+
+After that please install libraries required for development:
+
+.. code-block:: shell
+
+   $ pip install -r requirements/dev.txt
+
+Congratulations, you are ready to run the test suite!
+
+Run aiobotocore test suite
+--------------------------
+
+After all the preconditions are met you can run tests typing the next
+command:
+
+.. code-block:: shell
+
+   $ make test
+
+Run aiobotocore flake suite
+---------------------------
+
+We try our best to write code according to pep, it will allow you to check yourself.
+
+.. code-block:: shell
 
+   $ make flake
 
-After that please install libraries required for development::
+Run aiobotocore coverage suite
+------------------------------
 
-    $ pip install -r requirements-dev.txt
-    $ pip install -e .
+We are trying hard to have good test coverage; please don't make it worse.
 
-Congratulations, you are ready to run the test suite::
+.. code-block:: shell
 
-    $ make cov
+   $ make cov
 
-To run individual use following command::
+.. note::
 
-    $ py.test -sv tests/test_monitor.py -k test_name
+  If you want to use AWS within the test suite, you need create config ``~/.aws/credentials`` or the ``environment variable``
+
+  Then the test run will look like this:
+  .. code-block:: shell
+
+     $ make aws-cov
+     $ make aws-test
+
+
+Documentation
+-------------
+
+We encourage documentation improvements.
+
+Please before making a Pull Request about documentation changes run:
+
+.. code-block:: shell
+
+   $ make doc
+
+Once it finishes it will output the index html page
+``open file://`pwd`/docs/_build/html/index.html``.
+
+Go to the link and make sure your doc changes looks good.
+
+Changelog update
+----------------
+
+The ``CHANGES.rst`` contains information on changes.
+
+.. note::
+
+   If you know the release version and want to add a new commit before released out
+
+   Example::
+
+      0.1.0 (2018-09-01)
+      ^^^^^^^^^^^^^^^^^^
+
+      * Release & addition of changes file (Release-1.1.0)
+      * Name commit
+      * Name commit
+
+   If you set the date and version, it will be the last and will be released
+   Version must be raised in your last committee
```

Comment:
FYI, I've created templates for logging new issues and PRs.  Could you also in here update the template so the two match each other?